Tests/Fixture.swift

```diff
@@ -40,7 +40,7 @@ struct Fixture {
     
     /// The URL of the fixture within the test bundle.
     var URL: NSURL {
-        let bundle = NSBundle(identifier: "com.diephouse.matt.TentacleTests")!
+        let bundle = NSBundle(forClass: GitHubErrorTests.self)
```

Comment:
Aah, oops. I thought it was because the build was getting confused with the `TentacleTests` being the product name as well. Will update.